flink-connectors/flink-connector-aws-kinesis-data-streams/src/test/java/org/apache/flink/connectors/kinesis/testutils/KinesaliteContainer.java

```diff
@@ -45,33 +52,28 @@
 public class KinesaliteContainer extends GenericContainer<KinesaliteContainer> {
     private static final String ACCESS_KEY = "access key";
     private static final String SECRET_KEY = "secret key";
+    private static final int PORT = 4567;
+    private static final Region REGION = Region.US_EAST_1;
+    private static final String URL_FORMAT = "https://%s:%s";
 
     public KinesaliteContainer(DockerImageName imageName) {
         super(imageName);
 
-        withEnv(ACCESS_KEY_ENV_VAR, ACCESS_KEY);
-        withEnv(SECRET_KEY_ENV_VAR, ACCESS_KEY);
-        withExposedPorts(4567);
+        System.setProperty(SdkSystemSetting.CBOR_ENABLED.property(), "false");
```

Comment:
Same as above, can this be set on client instead?

Reply:
Removed. Thanks.